js/stack-trace/index.js

```diff
@@ -72,14 +73,20 @@ class WrappedCallSite {
   }
 }
 
-function getPrepareStackTrace (originalPrepareStackTrace) {
+function isValidFilenameDefault (fileName) {
+  return true
+}
+
+function getPrepareStackTrace (originalPrepareStackTrace, isValidFilename) {
   if (originalPrepareStackTrace && originalPrepareStackTrace[kSymbolPrepareStackTrace]) {
     return originalPrepareStackTrace
   }
 
+  isValidFilename = isValidFilename ?? isValidFilenameDefault
```

Comment:
Not supported in node 12:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
we can change it in the line 80
```
function getPrepareStackTrace (originalPrepareStackTrace, isValidFilename = isValidFilenameDefault) {
```